src/main/asciidoc/annotations.adoc

```diff
@@ -0,0 +1,20 @@
+= JSR-107(JCache) Annotations
+
+If you are familiar with Spring, you know that it provides annotations to assist in developing applications.  In regards to caching, Spring offers support for two sets of annotations that can be used to implement caching.  You have the original Spring annotations and the new JSR-107 annotations.  The original Spring annotations are available to use with Spring versions 3.1+, while the JSR-107 annotations are only available in Spring 4.1+.  In this example we are going to use the JSR-107 annotations.  Below I have listed the most commonly used JSR-107(JCache) annotations, with brief descriptions as well as links to their API's.
+
+* *@CacheDefaults* - allows configuration of defaults at the class level.  For instance, you can define a cache name at the class level and this will be used as the default.
+** http://ignite.apache.org/jcache/1.0.0/javadoc/javax/cache/annotation/CacheDefaults.html
```

Comment:
I don't think we want to refer to (reference) apache ignite .   this is gemfire re-packaged as OSS and has become a competitor.  You should be able to find the official version of these here in github at the official jsr107 project.